acm/src/main/java/org/literacybridge/acm/store/AudioItemIndex.java

```diff
@@ -380,6 +361,24 @@ private Playlist getPlaylist(final String uuid, final Playlist playlist) throws
         }
     }
 
+    private void loadPlaylistFromPostingList(LeafReader leafReader, String playlistUuid, Playlist.Builder playlistBuilder) throws IOException {
+        PostingsEnum tp = leafReader.postings(new Term(PLAYLISTS_FIELD, playlistUuid), PostingsEnum.PAYLOADS);
```

Comment:
Sorry, old habit. The class PostingsEnum used to be called TermPositions in Lucene. Using "tp" as variable name was very common. I renamed it to "postingsEnum". It's an iterator used here to iterate over all AudioItems belonging to the given Playlist.